### DISCUSSION TITLE
TTS 목소리가 하나로 고정됩니다

### DISCUSSION THREAD

**Discussion by NEFiNi** [Lastorder-DC/chatreader-kor#68] [문의]
윈도우 10 크롬브라우저를 사용하는데 !!tts uniq voice on 명령어를 입력해도 유저별로 목소리가 구분되지 않고 다른 채널의 채팅을 불러와도 같은 목소리로만 재생됩니다

**Comment by Lastorder-DC** [comment]
증상이 재현되지 않습니다.

1. 개인화 설정은 `!!tts uniq on`으로 켤 수 있습니다.
2. 해당 채널 스트리머 본인은 이 설정을 적용받지 않습니다.
3. 크롬 브라우저에서만 동작합니다. 엣지/웨일을 제외한 크롬 기반 브라우저에서는 정상 동작하지 않을 수 있습니다. 방송 프로그램 내장 브라우저가 대표적인 미지원 브라우저입니다.
